docs/command/mongocli.txt

```diff
@@ -49,6 +49,7 @@ Related Commands
 ----------------
 
 * :ref:`mongocli-atlas` - Atlas operations.
+* :ref:`mongocli-auth` - Manage the CLI's authentication state.
```

Comment:
Authenticate with MongoDB Atlas using the CLI.

Reply:
this "management" includes cloud manager, also this copy in particular was suggested by Melissa

Reply:
Let's keep Melissa's copy then!